### DISCUSSION TITLE
Incorrect package deploy to npm

### DISCUSSION THREAD

**Discussion by Shaquu** [material-table-core/core#9] [General]
Hi, 
package got release to npm without dist folder.
It is unusable `Module not found: Can't resolve '@material-table/core' `

**Comment by peacechen** [comment]
@oze4 
material-table was set up such that main points to the dist folder:
https://github.com/material-table-core/core/blob/master/package.json#L8

That folder is created during `npm run build` transpilation. Before publishing, always run `npm run build`, which should be added to the CI/CD script.
Alternatively, you could point main to `src/index.js` to avoid the build step. That may not play nice with projects that don't have all the necessary babel plugins. We need to test whether the code depends on experimental plugins.

**Reply by matthewoestreich** [reply]
I did build before publishing (dist folder was created and everything).

Give me a couple minutes and I'll check this out. 

Apologies.

**Comment by matthewoestreich** [comment]
@Shaquu @peacechen apologies this has been resolved.

**Reply by matthewoestreich** [reply]
@Shaquu @peacechen see here https://github.com/material-table-core/core/releases/tag/v2.1.4 and here https://github.com/material-table-core/core/issues/10 for more.

**Reply by matthewoestreich** [reply]
I tested `@material-table/core`, [`v2.1.4`](https://www.npmjs.com/package/@material-table/core) as well to make sure it's working (it is).

**Reply by peacechen** [reply]
Thanks @oze4 !

**Reply by thefat32** [reply]
@oze4 you can unpublish broken versions with npm unpublish since it has not been 48hs since deploy. If you need me to handle any task regarding npm or deploys just add me to material-table-core org!

**Reply by matthewoestreich** [reply]
@thefat32 invite has been sent. (to the email listed on your github profile)

**Comment by matthewoestreich** [comment]
@Domino987 @peacechen @thefat32 @Shaquu 

Please see [here](https://github.com/material-table-core/core/discussions/11). I have configured GH Actions to automatically publish NPM package, only if certain params are met..

**Reply by Shaquu** [reply]
You could make it work on new tag instead of commit message

**Reply by thefat32** [reply]
Great!

**Reply by matthewoestreich** [reply]
@Shaquu I'm down for whatever everyone feels is best. I honestly have no preference. If you prefer via tag, let's change it.
